tests/src/main/java/org/hibernate/beanvalidation/tck/tests/valueextraction/unwrapping/ValueExtractionUnwrappingTest.java

```diff
@@ -122,7 +121,9 @@ public void validate_wrapped_value_if_value_extractor_unwraps_by_default_for_gen
 	@SpecAssertion(section = Sections.CONSTRAINTDECLARATIONVALIDATIONPROCESS_VALIDATIONROUTINE_VALUEEXTRACTORRESOLUTION_IMPLICITUNWRAPPING, id = "a")
 	public void validate_wrapper_if_unwrapping_disabled_per_constraint_for_generic_container() {
 		Set<ConstraintViolation<WrapperWithDisabledUnwrapping>> constraintViolations = getValidatorWithValueExtractors().validate( new WrapperWithDisabledUnwrapping() );
-		assertNumberOfViolations( constraintViolations, 0 );
+		assertThat( constraintViolations ).containsOnlyViolations(
+				violationOf( Null.class ).withProperty( "integerWrapper" )
```

Comment:
Wouldn't this test make more sense if it was a constraint with an implicitly unwrapping value extractor? `@Null` would be applied to the wrapper anyways.